mkdocs.yml

```diff
@@ -136,3 +136,6 @@ nav:
       - vsftpd: apps/vsftpd.md
       - Watchtower: apps/watchtower.md
       - ZNC: apps/znc.md
+ 
+  - Docker Compose Overrides
```

Comment:
I think I'd like to just label this `Overrides` and then we can move the existing overrides page under this heading and rename the page to `Introduction` and leave it at the top of the list. It would also be good to leave behind a placeholder page that links to the new introduction page location.

Reply:
that sounds like a great idea. i'll fix it up.